homeassistant/components/edilkamin/entity.py

```diff
@@ -0,0 +1,80 @@
+"""Edilkamin integration entity."""
+from typing import Any, cast
+
+import edilkamin
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+
+from .const import LOGGER
+
+power_to_hvac = {
+    edilkamin.Power.OFF: HVACMode.OFF,
+    edilkamin.Power.ON: HVACMode.HEAT,
+}
+hvac_mode_to_power = {hvac: power for (power, hvac) in power_to_hvac.items()}
+
+
+class EdilkaminEntity(ClimateEntity):
```

Comment:
Please look into the [updated entity naming rules](https://developers.home-assistant.io/docs/core/entity#entity-naming) and set `has_entity_name` to `True`